src/solver/conv_hip_implicit_gemm_wrw_v4r4_xdlops_padded_gemm.cpp

```diff
@@ -0,0 +1,1106 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2020 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#include <miopen/solver.hpp>
+#include <miopen/conv/invokers/impl_gemm.hpp>
+#include <miopen/conv/wrw_invoke_params.hpp>
+#include <miopen/handle.hpp>
+#include <miopen/generic_search.hpp>
+#include <miopen/hip_build_utils.hpp>
+#include "implicitgemm_util.hpp"
+
+#include <miopen/stringutils.hpp>
+#include <miopen/tensor_ops.hpp>
+#include <miopen/implicitgemm_params.hpp>
+
+namespace miopen {
+namespace solver {
+
+PerformanceImplicitGemmWrwV4R4Xdlops_Padded_Gemm::PerformanceImplicitGemmWrwV4R4Xdlops_Padded_Gemm()
+    : PerformanceImplicitGemmWrwV4R4Xdlops_Padded_Gemm::
+          PerformanceImplicitGemmWrwV4R4Xdlops_Padded_Gemm(
+              64, 64, 2, 32, 32, 4, 16, 64, 16, false, false)
```

Comment:
Please add comment here that GemmMFactor GemmNFactor, GemmKFactor are fixed value at this moment.